whisper/__init__.py

```diff
@@ -6,6 +6,7 @@
 from typing import List, Optional, Union
 
 import torch
+from habana_frameworks.torch.utils.library_loader import load_habana_module
```

Comment:
Does this work on machines/containers without habana_frameworks package? Wondering if load_habana_module and wrap_in_hpu_graph can be imported only if `device=="hpu"`.

Reply:
Thanks, thats true, changed to be loaded on condition - https://github.com/BlueLabelLabs/whisper_fork/pull/2/files#diff-ccf3e38e2062ec2b77cbeda0fb709b927859e107a1d84463651fd53915182616R47